cmd/snap/cmd_quota.go

```diff
@@ -0,0 +1,116 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/strutil"
+)
+
+var shortQuotaHelp = i18n.G("Create, update or show quota group for a set of snaps")
+var longQuotaHelp = i18n.G(`
+The quota command creates, updates or shows quota groups for a a set of snaps.
+
+A quota group sets resource limits (currently maximum memory only) on the set of
+snaps that belong to it. Quotas groups are controlled by systemd slice units.
```

Comment:
I'm unsure whether we want to mention systemd here. cc @anonymouse64

Reply:
I think we should avoid mentioning systemd for now